fastats/core/ast_transforms/processor.py

```diff
@@ -22,55 +23,81 @@ def __init__(self, top_level_func, overrides, replaced, new_funcs=None):
         self._debug = self._overrides.get('debug')
 
     def process(self):
-        source = inspect.getsource(self.top_level_func)
-        tree = ast.parse(source)
+        try:
+            source = inspect.getsource(self.top_level_func)
+            tree = ast.parse(source)
+        except IndentationError:
```

Comment:
Where can this come from? `parse`?

Reply:
Yep, precisely. `parse` will raise an IndentationError if a nested function source is passed in